newroutes/sites.js

```diff
@@ -0,0 +1,73 @@
+const autoBind = require("auto-bind")
+const express = require("express")
+
+// Import middleware
+const { attachReadRouteHandlerWrapper } = require("@middleware/routeHandler")
+
+class SitesRouter {
+  constructor({ sitesService }) {
+    this.sitesService = sitesService
+    // We need to bind all methods because we don't invoke them from the class directly
+    autoBind(this)
+  }
+
+  async getSites(req, res) {
+    const { accessToken } = req
+    const siteNames = await this.sitesService.getSites({ accessToken })
+    return res.status(200).json({ siteNames })
+  }
+
+  async checkHasAccess(req, res) {
+    const { accessToken, userId } = req
+    const { siteName } = req.params
+
+    await this.sitesService.checkHasAccess(
+      {
+        accessToken,
+        siteName,
+      },
+      { userId }
+    )
+    return res.status(200).send("OK")
```

Comment:
https://github.com/isomerpages/isomercms-backend/pull/341/commits/e64781021882ea0289420e5ee8d68a4015f14341

Reply:
^this commit got lost, unsure if there are others. i'm fine iwth not doing this as it's relatively minor

Reply:
The commit was preserved as 9b05ff7 - rebasing seems to overwrite the original commit hash! I did reintroduce the `.send("OK")` in a later commit because one of the tests was failing, and elected to use `.send("OK")` to keep it in line with how we return responses in the rest of our routers